### ISSUE TITLE
Importing the same CRDs in two subcharts results in an error

### ISSUE THREAD

**Issue by phoenix-bjoern** [helm/helm#12513]
When importing the same CRDs in two subcharts and run `helm template --output-dir` and error occurs.

Steps to reproduce:
1. Create a Chart.yaml in an empty directory:
```
apiVersion: v2
name: snapshot-test
type: application
version: 0.1.0
appVersion: 0.1.0
dependencies:
  - name: "snapshot-controller"
    version: "1.9.1"
    repository: https://piraeus.io/helm-charts/
  - name: "snapshot-validation-webhook"
    version: "1.8.2"
    repository: https://piraeus.io/helm-charts/
```

2. Run `helm dep up` to pull the dependencies.
3. Generate the template and write them to an output dir:
```
$ helm template --include-crds --output-dir test .
wrote test/snapshot-test/charts/snapshot-controller/crds/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml
wrote test/snapshot-test/charts/snapshot-controller/crds/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
wrote test/snapshot-test/charts/snapshot-controller/crds/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
wrote test/snapshot-test/charts/snapshot-controller/crds/snapshot.storage.k8s.io_volumesnapshots.yaml
Error: open test/snapshot-test/charts/snapshot-validation-webhook/crds/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml: no such file or directory
```

Without the `--ouput-dir` parameter the command renders the resources, CRDs only appear twice in the output.

While it probably doesn't make sense to have the same CRDs in both charts, I think Helm shouldn't exit with an error.

**Comment by z4ce** [comment]
Works for me on MacOS with Helm v1.13.1

```
╰─ helm template --include-crds . --output-dir test
wrote test/snapshot-test/charts/snapshot-controller/crds/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml
wrote test/snapshot-test/charts/snapshot-controller/crds/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
wrote test/snapshot-test/charts/snapshot-controller/crds/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
wrote test/snapshot-test/charts/snapshot-controller/crds/snapshot.storage.k8s.io_volumesnapshots.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/crds/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/crds/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/crds/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/crds/snapshot.storage.k8s.io_volumesnapshots.yaml
wrote test/snapshot-test/charts/snapshot-controller/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/webhook.yaml
wrote test/snapshot-test/charts/snapshot-controller/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-controller/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-controller/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-controller/templates/serviceaccount.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/service.yaml
wrote test/snapshot-test/charts/snapshot-controller/templates/deployment.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/deployment.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/webhook.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/webhook.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/tests/test-invalid-body.yaml
wrote test/snapshot-test/charts/snapshot-validation-webhook/templates/tests/test-valid-body.yaml
```

**Comment by phoenix-bjoern** [comment]
Upgrade Helm from 3.12.2 -> 3.13.1, which actually resolved the issue. Thanks @z4ce for cross-checking!

**Comment by phoenix-bjoern** [comment]
Just for the record: The issue is resolved in the [Helm 3.13.0 release.](https://github.com/helm/helm/releases/tag/v3.13.0) The change list is long, so not sure which change actually resolved the problem.

To test the behavior of different Helm version use a container with the version tag:
```
# shows the error
docker run -ti --rm -v $(pwd):/apps -w /apps alpine/helm:3.12.3 template --include-crds --output-dir test .
# works with 3.13.0
docker run -ti --rm -v $(pwd):/apps -w /apps alpine/helm:3.13.0 template --include-crds --output-dir test .
```
